graphistry/dgl_utils.py

```diff
@@ -295,85 +306,137 @@ def _convert_edgeDF_to_DGL(self, node_column: Optional[str] = None, weight_colum
         )
         res._index_to_entity = {k: v for v, k in res._entity_to_index.items()}
         # this is a sanity check after _remove_edges_not_in_nodes
-        res._check_nodes_lineup_with_edges(node_column)
+        res._check_nodes_lineup_with_edges()
         return res
 
-
     def _featurize_nodes_to_dgl(
         self,
         res,
         X: pd.DataFrame,
         y: pd.DataFrame,
         use_scaler: str = None,
-        #refeaturize: bool = False,
+        feature_engine: FeatureEngine = "auto",
     ):
         logger.info("Running Node Featurization for DGL Graph")
+        logger.debug(f"=*=*=Input shapes are data: {X.shape}, target: {y.shape}")
+
+        X_enc, y_enc, res = res._featurize_or_get_nodes_dataframe_if_X_is_None(
+            X=X,
+            y=y,
+            use_scaler=use_scaler,
+            feature_engine=resolve_feature_engine(feature_engine),
+        )
+
+        logger.debug(
+            f"=*=*=Encoded Node shapes are data: {X_enc.shape}, target: {y_enc.shape}"
```

Comment:
do not use `f'{...}` during logging, should be `logger.debug("some string: %s", some_var)` : that way the string interpolation gets skipped when the log mode is off